relay/kusama/src/lib.rs

```diff
@@ -764,6 +780,23 @@ impl pallet_treasury::Config for Runtime {
 	type WeightInfo = weights::pallet_treasury::WeightInfo<Runtime>;
 	type SpendFunds = Bounties;
 	type SpendOrigin = TreasurySpender;
+	type AssetKind = VersionedLocatableAsset;
```

Comment:
Not sure if it needs to be added to Polkadot/Kusama. @muharem can you, please, confirm that both Polkadot and Kusama need those changes - https://github.com/paritytech/polkadot-sdk/pull/1333 ? Thank you!

Reply:
I would rase a separate PR for this, to provide an appropriate description and have it reviewed. You can set up this in a restrictive way, disabling the new feature.

Reply:
Ok, thank you! I'll remove it from this PR then

Reply:
Removed in https://github.com/polkadot-fellows/runtimes/pull/87/commits/5cfef32ef7525812d0522c376a43430cb9945abb. Had to introduce this `NeverLookup` struct to disable functionality in production. IIUC it is enough - `spend` would fail and all other new methods would fail because there's no corresponding entry in the `Spends` map